src/cargo.md

```diff
@@ -0,0 +1,9 @@
+# Cargo
+
+`cargo` là công cụ quản lý gói chính thức của Rust. Nó có rất nhiều tính năng thực sự để giúp cải thiện chất lượng và tốc độ viết mã của nhà phát triển hữu ích! Bao gồm các chức năng:
+
+- Quản lý các phụ thuộc và tích hợp với [crates.io](https://crates.io) (nơi đăng ký các gói chính thức của Rust)
+- Sử dụng unit tests
```

Comment:
2 từ gốc là
Awareness of unit tests
Awareness of benchmarks
Chưa biết dùng từ nào cho thích hợp.
Anh em cho ý kiến dùng dịch thế nào cho phù hợp chỗ này nhé

Reply:
# Tự động phát hiện và thực thi unit tests, sử dụng câu lệnh `cargo test`

Reply:
@SkyTik dịch vậy có nhiều quá không nhỉ